streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -217,4 +224,78 @@ public void setUncaughtExceptionHandler(Thread.UncaughtExceptionHandler eh) {
             thread.setUncaughtExceptionHandler(eh);
     }
 
+
+    /**
+     * Find all of the instances of {@link KafkaStreamsInstance} in a {@link KafkaStreams application}
```

Comment:
Nit. For nice JavaDoc formatting we need either an empty line in between or add `<br />` or `<p>`. Otherwise, everything will be in a single paragraph, "hiding" *Note:*

Same for other JavaDocs further down containing "Note:"